libpq/libpq.go

```diff
@@ -0,0 +1,12 @@
+package libpq
+
+import (
+  "fmt"
+  psh "github.com/platformsh/config-reader-go/v2"
+)
+
+// lib/pq requires a specfic connection string to connect to PostgreSQL.
+func FormattedCredentials(creds psh.Credential) (string, error) {
+  formatted := fmt.Sprintf("host=%s port=%d user=%s " + "password=%s dbname=%s sslmode=disable", creds.Host, creds.Port, creds.Username, creds.Password, creds.Path)
```

Comment:
Splitting the line on "format string" and "other" is fine.  That's a logical place to split.  But why the two format strings +-ed together?  That's what I don't grok.